python/ray/_private/runtime_env/working_dir.py

```diff
@@ -54,7 +54,11 @@ def upload_working_dir_if_needed(
         protocol, path = None, None
 
     if protocol is not None:
-        if protocol in Protocol.remote_protocols() and not path.endswith(".zip"):
+        if (
+            protocol in Protocol.remote_protocols()
+            and not path.endswith(".zip")
+            and not path.endswith(".whl")
```

Comment:
Why do we need to support whl for working dir?

Reply:
missed this, shld've not supported